gen/calc.py

```diff
@@ -104,8 +104,15 @@ def validate_ipv4_addresses(ips: list):
 
 
 def validate_absolute_path(path):
-    if not path.startswith('/'):
-        raise AssertionError('Must be an absolute filesystem path starting with /')
+    if not os.path.isabs(path):
+        raise AssertionError('Must be an absolute filesystem path')
+
+
+def calculate_json_escape(value):
+    # calculate the value to be interpolated into a JSON string
```

Comment:
maybe a bit more doc is needed here. This translates to json but if the object is plain string its translated back into plain (escaped) string?

Reply:
It only works on strings, since the assert will fail if it is anything else. The point is to allow a template to contain `{"attr": "something {{ myvar }} else"}`

Reply:
Since a test failed and needed to be fixed, I have also updated the comment to be a bit more clear.